src/hooks/useLoadDisciplines.ts

```diff
@@ -21,9 +21,15 @@ const formatServerResponse = (serverResponse: ReturnType<ServerResponse[]>): Ret
   return { ...serverResponse, data: formattedServerResponse }
 }
 
-export const useLoadDisciplines = (parent: DisciplineType | null): ReturnType<DisciplineType[]> => {
+export const useLoadDisciplines = (
+  parent: DisciplineType | null,
+  customDisciplines?: string[]
+): ReturnType<DisciplineType[]> => {
   const prefix = parent?.isLeaf ? ENDPOINTS.trainingSet : ENDPOINTS.disciplines
-  const suffix = parent?.id ?? ''
+  const customDisciplinesSuffix = customDisciplines ? customDisciplines.join('&') : ''
+  let suffix = parent?.id ?? customDisciplinesSuffix
+  console.log(`${prefix}/${suffix}`)
+  suffix = parent?.id ?? '' // TODO LUN-190 remove when api is ready
   const disciplines = useLoadFromEndpoint<ServerResponse[]>(`${prefix}/${suffix}`)
```

Comment:
Getting an eslint error that `prefix` is of type any. However, the checks succeeded. Is linting not working again in the CI?